pyfar/classes/coordinates.py

```diff
@@ -1380,10 +1380,309 @@ def show(self, mask=None, **kwargs):
             colors[mask] = pf.plot.color('r')
             pf.plot.scatter(self, c=colors.flatten(), **kwargs)
 
+    def find_nearest(self, find, k=1, distance_measure='euclidean'):
+        """
+        Find the k nearest coordinates points.
+
+        Parameters
+        ----------
+        find : pf.Coordinates
+            Coordinates to which the nearest neighbors are searched.
+        k : int, optional
+            Number of points to return. k must be > 0. The default is ``1``.
+        distance_measure : string, optional
+            ``'euclidean'``
+                Distance is determined by the euclidean distance i meters
+                between the points. This is default.
+            ``'spherical'``
+                Distance is determined by the great-circle distance in meters
+                between the points.
+            ``'angular'``
+                distance is determined by the angles in radiant.
```

Comment:
This I find confusing. Do I get it right that both use the great circle distance, i.e., the smallest distance between two points on a sphere but *spherical* expects it in meters and *angular* in radians? If yes, I think we should discuss better names and find a better description. If not - I'm lost.